examples/playground/overlays/example-append-array-item/append-array-item.yml

```diff
@@ -3,13 +3,13 @@
 #@overlay/match by=overlay.all
 ---
 awards:
-#@overlay/append
 - Ovid Prize
 
 #! Each edit requires two parts:
 #! 1. which item(s) in the collection to target (match)
 #! 2. what operation to perform
 
-#! `@overlay/append`:
+#! Omitting an annotation on the array item implies a
+#! default 'append' operation:
```

Comment:
this is no longer true per changes in this PR. it's still a merge operation that does not match any existing node, so append happens